config.example.yaml

```diff
@@ -60,6 +60,11 @@ templates:
             version: 1.0.0
             type: file
 
+      /{api:sys}/{module:key_rev_value}: &wp/sys/key_value
```

Comment:
The `wp/sys/key_value` achor name is already in use. We should probably rename that to `wp/sys/key_rev_value` for consistency's sake, anyway.